testsuite/MDAnalysisTests/topology/test_pdb.py

```diff
@@ -233,6 +235,7 @@ def test_PDB_hex():
     assert u.atoms[3].id == 100002
     assert u.atoms[4].id == 100003
 
+@pytest.mark.filterwarnings("ignore::UserWarning")
```

Comment:
Sorry, I failed to reply to this. The no_symbols is essential in case there is an element which not has been detected (.i.e. is essentially missing). It basically returns an empty string in case the element provided is non-physical or missing, which was the consensus. Does it sound reasonable?

Reply:
I'm not sure you replied to the right comment here. If you're talking about `no_symbols` in `guess_atom_element`, `no_symbols` is just the name without any *, +, or - symbols in it. So `no_symbols` for 'AO*5' would be 'AO5', rather than the empty string.